search.go

```diff
@@ -131,11 +169,15 @@ func (req *SearchRequest) Map() map[string]interface{} {
 		m["size"] = *req.size
 	}
 	if len(req.sort) > 0 {
-		sortMaps := make([]map[string]interface{}, 0, len(req.sort))
+		sortSlice := make([]interface{}, 0, len(req.sort))
```

Comment:
this can be reverted as well since rawSort is not required. we can achieve same functionality with regular field sortparams as well